examples/kubernetes/README.md

```diff
@@ -0,0 +1,146 @@
+# :dango: Pipeline Orchestration on Kubernetes
+
+This example will demonstrate how to orchestrate pipelines using the ZenML Kubernetes-native orchestrator. We will build a simple pipeline consisting of four steps and orchestrate it in an Amazon EKS Kubernetes cluster.
+
+## :heavy_check_mark: Requirements
+
+In order to follow this tutorial you need to first spin up an Amazon EKS cluster, an ECR container registry, as well as a S3 bucket 
+for artifact storage. 
+
+Furthermore, you need to have kubectl and Docker installed locally.
+
+### Setup and Register Kubernetes Orchestrator on EKS
+After spinning up your EKS cluster, run the following command to grant access to your local kubectl.
+
+```bash
+AWS_REGION = "eu-central-1"
+AWS_EKS_CLUSTER = "native-kubernetes-orchestrator"
+KUBE_CONTEXT = "zenml-kubernetes" 
+aws eks --region {AWS_REGION} update-kubeconfig --name {AWS_EKS_CLUSTER} --alias {KUBE_CONTEXT}
```

Comment:
In most other places we use pointy brackets to show which placeholders the user needs to specify.

Reply:
Great points. I changed all steps to only mention AWS as an example and replaced curly brackets with angular ones.